README.md

```diff
@@ -21,67 +21,163 @@ e.g.
 to load your operator bundle in OpenShift.
 
-**Create your own index image**
 
-Requires [opm](https://github.com/operator-framework/operator-registry/releases) v1.15.3+ -
+Requires [opm](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest/)
 
-```bash
-USERNAME=tchughesiv
-VERSION=$(go run getversion.go -csv)
-IMAGE=quay.io/${USERNAME}/rhpam-operator-bundle
-BUNDLE=${IMAGE}:${VERSION}
+Requires [podman](https://podman.io/)
+
+Requires [cekit](https://github.com/cekit/cekit)
+
+**Create your own Bundle**
+Remove the following line from deploy/olm-catalog/dev/<last-version>/manifest/businessautomation-operator.clusterserviceversion.yaml
+```console
+replaces: businessautomation-operator.<last-version>
+```
+
+Activate Cekit and run the following command
+```console
 $ make bundle-dev
+```
+
+the last log line is something like this:
+```console
+INFO  Image built and available under following tags: quay.io/<your_quay_username>/rhpam-operator-bundle:7.12.1, quay.io/${USERNAME}/rhpam-operator-bundle:latest
+```
+**Push the bundle on the container registry**
+
+VERSION=$(go run getversion.go)
+
+USERNAME=<your_quay_username>
+```console
+$ docker push quay.io/${USERNAME}/rhpam-operator-bundle:${VERSION}
+```
+
+#### Build the index
+```console
+opm index add --bundles quay.io/${USERNAME}/rhpam-operator-bundle:${VERSION} --tag quay.io/${USERNAME}/rhpam-operator--index:${VERSION} --permissive
+```
+#### Push the index on the container registry
+Podman images
+```console
+podman push quay.io/${USERNAME}/rhpam-operator--index:${VERSION}
+```
 
+#### Disable default catalog sources on Openshift
+To test your Operator, with bundle and index you need to disable the default source like the operator hub
+```console
+oc patch OperatorHub cluster --type json -p '[{"op": "add", "path": "/spec/disableAllDefaultSources", "value": true}]'
+```
 
-$ docker push ${BUNDLE}
-BUNDLE_DIGEST=$(docker inspect --format='{{index .RepoDigests 0}}' ${BUNDLE})
-INDEX_VERSION=v4.7
-INDEX_IMAGE=quay.io/${USERNAME}/ba-operator-index:${INDEX_VERSION}
-INDEX_FROM=${INDEX_IMAGE}_$(go run getversion.go -csvPrior)
-INDEX_TO=${INDEX_IMAGE}_${VERSION}
+#### catalog-source.yaml
+```console
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: xxxxxname
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: xxxxximage
+  displayName: My Operator Catalog
+  publisher: grpc
+```
 
-$ opm index add -c docker --bundles ${BUNDLE_DIGEST} --from-index ${INDEX_FROM} --tag ${INDEX_TO}
+Choose a CATALOG_SOURCE_NAME something like "max-operator-manifests"
 
-$ docker push ${INDEX_TO}
+```console
+sed -i "s/xxxxxname/$CATALOG_SOURCE_NAME/" catalog-source.yaml
+```
+```console
+sed -i "s/xxxxximage/$(echo $INDEXING_IMAGE | sed "s.\/.\\\/.g")/" catalog-source.yaml
+```
```

Comment:
I wouldn't keep there these sed commands.
Instead, it would be better to have there the catalog-source.yaml with already replaced values. There can be additional comments to this file, what is needed to be replaced there - the image reference.